draft-ietf-quic-version-negotiation.md

```diff
@@ -241,23 +269,20 @@ Client-Sent Other Versions:
 
 : When sent by a client, the `Other Versions` field lists all the versions that
```

Comment:
I just noticed this editorial convention: I think that you should avoid using backticks for field names.  The QUIC spec doesn't use them.  (One nasty consequence of using them is that you get scare quotes in the text rendering.)